### ISSUE TITLE
2.5gb on small project

### ISSUE THREAD

**Issue by edunsouza** [Microsoft/vscode#62922]
I have a small project of Natural Language Processing and there are 2 files with 37k lines, which contains the datasets. The problem occurs even when i'm not using those files. Not even opening the files. I have to use other editor to run "node index.js" -my main fail.

I dont know how the problem starts, but when i open the vscode editor, it goes up to 2.5gb of ram on the startup. I dont know what is happening.

**Comment by chrmarti** [comment]
/needsMoreInfo

What is the file type of the data files?

**Comment by edunsouza** [comment]
@chrmarti i uninstalled some angular, ionic and cordova extensions and now vscode seems to be normal again!! Thank you so much.

I dont know exactly which one of the 6 - 7 extensions was eating the ram, but they were all related to ionic, cordova and angular tools.

Sorry for not helping enough.

**Comment by chrmarti** [comment]
Does it reproduce when starting with `code --disable-extensions`?

**Comment by edunsouza** [comment]
Yes, it does. I found out that the process “ts/js language features” is the
responsible for the 2.5gb usage. When i manually disabled the feature, it
starts to work normally. But only without JS intellisense features, and
thats sad :/

Em ter, 13 de nov de 2018 às 07:33, Christof Marti <notifications@github.com>
escreveu:

**Comment by mjbvz** [comment]
@edunsouza Can you share those source files?

**Comment by edunsouza** [comment]
@mjbvz of course I can! And I'll be happy if I can help to find out the problem.

The files are available on the repository: https://github.com/edunsouza/communicator
